near-sdk/src/json_types/account.rs

```diff
@@ -1,21 +1,31 @@
 use borsh::{BorshDeserialize, BorshSerialize};
-use serde::Serialize;
-use std::convert::{TryFrom, TryInto};
+use serde::{de, Deserialize, Serialize};
 use std::fmt;
+use std::{convert::TryFrom, str::FromStr};
```

Comment:
Nit, but let's keep uses separate, to be consistent with the rest of the code. z